### DISCUSSION TITLE
Explain where the name comes from

### DISCUSSION THREAD

**Discussion by AndreasBergmeier6176** [HermesGermany/galapagos#215] [Ideas]
We were wondering :)

**Comment by albrechtflo-hg** [comment]
Hi Andreas,

The first core feature was the JSON Schema Registry included in Galapagos. Its compatibility algorithm was very important for us to enable decoupled schema changes (where the consumer application does not have to be adjusted immediately) - this is called "schema evolution" and is no new idea, but can be found all over the web. So we took the "Evolution" together with the widespread usage of island names in the IT World... and voila :-)
